cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -37,16 +13,43 @@
 import no.unit.nva.model.contexttypes.Book;
 import no.unit.nva.model.contexttypes.Chapter;
 import no.unit.nva.model.contexttypes.Degree;
-import no.unit.nva.model.contexttypes.Journal;
 import no.unit.nva.model.contexttypes.PublicationContext;
 import no.unit.nva.model.contexttypes.Report;
+import no.unit.nva.model.contexttypes.UnconfirmedJournal;
+import no.unit.nva.model.contexttypes.UnconfirmedPublisher;
 import no.unit.nva.model.exceptions.InvalidIsbnException;
 import no.unit.nva.model.exceptions.InvalidIssnException;
+import no.unit.nva.model.exceptions.InvalidUnconfirmedSeriesException;
 import no.unit.nva.model.instancetypes.PublicationInstance;
 import no.unit.nva.model.pages.Pages;
 import nva.commons.core.attempt.Try;
 import nva.commons.core.language.LanguageMapper;
 
+import java.net.URI;
```

Comment:
fix the intellij idea rules for the static imports to be on top of the normal imports. Or in general we have to agree to a rule about this. It would make the PRs much smaller